### PR TITLE
Upgrade to sailor 2.6.18

### PR DESCRIPTION
**Pull request by denyshld**
Remove sensitive logs

### CODE REVIEW

File: lib/actions/attachmentToJson.js [hunk 1]
```diff
@@ -13,12 +13,12 @@ function checkFileName(self, fileName, pattern) {
...
 
   if (!pattern.test(fileName)) {
-    self.logger.debug('%s does not match pattern: \'%s\'', fileName, pattern);
+    self.logger.debug('Filename does not match pattern');
     return false;
   }
 
   if (fileName.split('.').pop() !== 'xml') {
-    self.logger.debug('%s is not .xml file: ', fileName);
+    self.logger.debug('Provided is not .xml file: ', fileName);
```
Comment (shulkaolka) [inline lib/actions/attachmentToJson.js:21]:
```suggestion
    self.logger.debug('Provided fileName is not .xml file');
```

File: lib/actions/attachmentToJson.js [hunk 2]
```diff
@@ -40,14 +40,14 @@ module.exports.process = async function processAction(msg, cfg) {
...
-    self.logger.info('Processing attachment=%s', fileName);
+    self.logger.info('Processing attachment');
 
     if (checkFileName(self, fileName, pattern)) {
       if (fileSize === undefined || fileSize < MAX_FILE_SIZE) {
         // eslint-disable-next-line no-await-in-loop
         const response = await new AttachmentProcessor().getAttachment(attachment.url, 'arraybuffer');
 
-        this.logger.debug(`For filename ${fileName} Response status: ${response.status} Response headers: ${JSON.stringify(response.headers)}`);
+        this.logger.debug(`For filename ${fileName} Response status: ${response.status}`);
```
Comment (shulkaolka) [inline lib/actions/attachmentToJson.js:50]:
```suggestion
        this.logger.debug(`For provided filename response status: ${response.status}`);
```

File: lib/actions/attachmentToJson.js [hunk 3]
```diff
@@ -61,12 +61,11 @@ module.exports.process = async function processAction(msg, cfg) {
           throw new Error(`Empty attachment received for file ${fileName}`);
         }
 
-        this.logger.debug(`For filename ${fileName} Response Body string: ${responseBodyString}`);
         fileSize = sizeof(responseBodyString);
 
         if (fileSize < MAX_FILE_SIZE) {
           const returnMsg = await xml2Json.process(this, responseBodyString);
-          this.logger.debug(`For filename ${fileName} Result of XML Conversion: ${JSON.stringify(returnMsg)}`);
+          this.logger.debug(`For filename ${fileName} Result of XML Conversed`);
```
Comment (shulkaolka) [inline lib/actions/attachmentToJson.js:68]:
```suggestion
          this.logger.debug(`Result of XML Conversed`);
```

File: lib/actions/jsonToXml.js [hunk 4]
```diff
@@ -52,7 +52,7 @@ module.exports.process = async function process(msg, cfg) {
   const attachmentProcessor = new AttachmentProcessor();
   const uploadResult = await attachmentProcessor.uploadAttachment(xmlString);
   const attachmentUrl = uploadResult.config.url;
-  this.logger.info(`Successfully created attachment at ${attachmentUrl}`);
+  this.logger.info('Successfully created attachment');
```
Comment (shulkaolka) [inline lib/actions/jsonToXml.js:55]:
```suggestion
  this.logger.info('Attachment created successfully');
```

**Review by shulkaolka — CHANGES_REQUESTED** [review]

File: package.json [hunk 5]
```diff
@@ -32,7 +32,7 @@
     "bluebird": "3.5.3",
     "co": "4.6.0",
     "elasticio-node": "0.0.9",
-    "elasticio-sailor-nodejs": "2.6.7",
+    "elasticio-sailor-nodejs": "2.6.18-dev",
```
Comment (A3a3e1) [inline package.json:35]:
```suggestion
    "elasticio-sailor-nodejs": "2.6.18",
```

File: lib/actions/attachmentToJson.js (hunk 1 above)
Comment (denyshld) [reply lib/actions/attachmentToJson.js]: done

File: lib/actions/attachmentToJson.js (hunk 2 above)
Comment (denyshld) [reply lib/actions/attachmentToJson.js]: done

File: lib/actions/attachmentToJson.js (hunk 3 above)
Comment (denyshld) [reply lib/actions/attachmentToJson.js]: done

File: lib/actions/jsonToXml.js (hunk 4 above)
Comment (denyshld) [reply lib/actions/jsonToXml.js]: done

File: package.json (hunk 5 above)
Comment (denyshld) [reply package.json]: Done

**Review by shulkaolka — CHANGES_REQUESTED** [review]
Please, update the following libs to the latest version:
component-commons-library
lodash

**Review by shulkaolka — APPROVED** [review]

**Review by A3a3e1 — APPROVED** [review]

**Pull request merged**